cuckoo_time_translator_algorithms/CMakeLists.txt

```
@@ -45,7 +45,7 @@ cs_add_library(${PROJECT_NAME}
 target_link_libraries(${PROJECT_NAME} ${console_bridge_LIBRARIES})
 
 cs_install()
-cs_export(INCLUDE_DIRS ${Eigen_INCLUDE_DIRS})
+cs_export(INCLUDE_DIRS ${Eigen3_INCLUDE_DIRS})
```

Here we need capital then of course as well.

```suggestion
cs_export(INCLUDE_DIRS ${EIGEN3_INCLUDE_DIRS})
```